### PR TITLE
Update README.md file

### PR DESCRIPTION
**Pull request by savitaashture**
Signed-off-by: Savita Ashture sashture@redhat.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -330,7 +330,9 @@ specified in the pipeline.
 
 First, you should create a number of `PipelineResources` that contain the specifics of the git repository and image registry to be used in the pipeline during execution. Expectedly, these are also reusable across multiple pipelines.
 
-The following `PipelineResource` defines the git repository for the frontend application:
+Following **PipelineResource** defines
```
Comment (chmouel) [inline README.md]: `The following` isnt it ?

File: README.md [hunk 2]
```diff
@@ -358,7 +360,7 @@ spec:
       value: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/ui:latest
 ```
 
-And the following `PipelineResource` defines the git repository for the backend application:
+The git repository for the backend application:
```
Comment (chmouel) [inline README.md]: why did you remove the info about PipelineResource?

**Review by chmouel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (savitaashture) [reply README.md]: Because every where PipelineResource was repeating so just added `PipelineResource` in the first place followed by explanation 

Reverted back to original

**Comment by savitaashture** [comment]
closing PR
As changes are handled as part of [PR](https://github.com/openshift/pipelines-tutorial/pull/64)
